zboxcore/sdk/uploadworker_test.go

```diff
@@ -63,3 +66,13 @@ func TestNumBlobbersRequiredZero(t *testing.T) {
 		t.Errorf("IsFullConsensusSupported() = %v, want %v", false, true)
 	}
 }
+
+func TestMarketplaceApi(t *testing.T) {
+	mnemonic, err := zboxutil.GetMarketplaceMnemonic("http://localhost:5051")
+	if err != nil {
+		fmt.Println("Got error", err)
```

Comment:
Please don't write output in unit tests.

Reply:
@Sriep Okay